### PR TITLE
Added sample codes to API:Contributors

### PR DESCRIPTION
**Pull request by tybanash**
Please review @brendajerop. Thank you

### CODE REVIEW

**Comment by brendajerop** [comment]
@tybanash Thank you for working on this!
A few initial comments:
* Since API:Contributors is a GET request, the code should be auto-generated. How you do that is by adding its JSON schema to [modules.json](https://github.com/wikimedia/mediawiki-api-demos/blob/master/modules.json), then run `python autogenerator.py` to autogenerate the code. Please see the last statement in [the contributing guidelines](https://github.com/wikimedia/mediawiki-api-demos/blob/master/contributing.md).
* Please provide a link to the sandbox page for this module.
* In your commit, make sure you add API:Contributors to the readme files of Python, PHP and JavaScript folders.

Feel free to ask if any of this is not clear :)

**Comment by tybanash** [comment]
@brendajerop  Thank you for the feedback. 
I just checked the last statement of the guidelines as you advised. Please can you explain this point. Its not clear.
$ python autogenerator.py
Make desired changes to the newly generated file(s)
Does this mean after running the above command and auto-generating the codes i still have to make changes to the code?
Also, they spoke only about autogenerating JS and python codes. Does this mean when working on a get request i don't need to bother about the php code?

**Comment by tybanash** [comment]
@brendajerop Thank you for the feedback. I just read the last statement of the guidelines as you advised. I'm not clear with this point. 
$ python autogenerator.py
Make desired changes to the newly generated file(s)
Please, after running the above command and the JS and python codes have been auto-generated, does this mean i still have to make changes on them?
Also they spoke only abt JS and python codes. Does this mean for get requests i don't need to bother about the php sample code?

**Comment by brendajerop** [comment]
@tybanash The code for PHP is autogenerated as well. I'll update the readme to reflect that.
If the example you are working on requires further processing of the data received from the request, then you may make changes to the code. If not, you may leave it as it is.

**Comment by tybanash** [comment]
@brendajerop Thank you for the clarification

**Comment by tybanash** [comment]
@brendajerop I have sent a new commit. Please review. This is the link of my sandbox page of API:Contributors. Thank you.

**Comment by tybanash** [comment]
@brendajerop I have sent a new commit. Please review. This is the link of my sandbox page of API:Contributors https://www.mediawiki.org/wiki/User:Awah_Nadege/Sandbox/API:Contributors . Thank you.

**Comment by brendajerop** [comment]
@tybanash Please resolve the merge conflict

**Comment by tybanash** [comment]
I did. I sent another commit after I noticed it yestererday. Please can you check the third commit @brendajerop

**Comment by brendajerop** [comment]
@tybanash I've seen the commit but there's still a conflict :)

**Comment by tybanash** [comment]
conflict issue
conflict 1
@brendajerop I just checked for the conflict using git bash and my github page but its written the branch is up to date. Please what other method can i use to resolve or can i send a different pull request?

File: javascript/contributors.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+/*
+    contributors.js
+
+    MediaWiki API Demos
+    Demo of `Contributors` module: Get request to list all logged-in contributors and count of anonymous contributors to a page.
+
+    MIT License
+*/
+
+const fetch = require('node-fetch');
```
Comment (brendajerop) [inline javascript/contributors.js]: You may remove this line for consistency with the other code samples

File: modules.json [hunk 2]
```diff
@@ -570,5 +570,16 @@
...
+    },
+    {
+        "filename": "contributors",
+        "docstring": "Demo of `Contributors` module: Get request to list all logged-in contributors and count of anonymous contributors to a page.",
+        "endpoint": "https://en.wikipedia.org/w/api.php",
+        "params": {
+            "action": "query",
+            "titles": "MediaWiki",
+            "prop": "contributors",
+			"format": "json"
```
Comment (brendajerop) [inline modules.json]: There's an inconsistency in the indentation

**Comment by brendajerop** [comment]
@tybanash Try rebasing your changes on top of the last commit in the master branch:)

**Comment by brendajerop** [comment]
@tybanash I've left feedback on your sandbox page here: https://www.mediawiki.org/wiki/User_talk:Awah_Nadege/Sandbox/API:Contributors

**Comment by tybanash** [comment]
Thank you for the feedback @brendajerop , will work on them.

**Comment by tybanash** [comment]
@brendajerop please can you check the second commit i sent on Oct 16th (dbeebcc), i renamed from contributors to get_contributors. I apologise for sending in so many commits, this is my first time using the rebase command.
Concerning the indentation of my code in modules.json, please look at this screenshot of my text editor
. It indents properly in my text editor. I dont know why when i push to github the indentation changes.

**Comment by tybanash** [comment]
@brendajerop. I have also updated my sandbox. Please review https://www.mediawiki.org/wiki/User:Awah_Nadege/Sandbox/API:Contributors

**Comment by tybanash** [comment]
I removed the contributors files. I tried rebasing again to verify this.  When i run git rm javascript/contributors.js for example, the result is 'javascript/contributors.js' did not match any files. Could you please have a look at the screenshot 
rebase
Please what do you suggest I do?

**Comment by brendajerop** [comment]
@tybanash I've merged your code but made a few changes to clean up your git history since your code was essentially correct :)

**Comment by brendajerop** [comment]
@tybanash  Ping me when you're done making the changes I requested in your sandbox :)

**Comment by tybanash** [comment]
Thank you @brendajerop . I have made the changes in my sandbox.

**Comment by tybanash** [comment]
Can I go ahead to merge the contents of my sandbox to the API page @brendajerop ?

**Pull request merged**